src/Main.cpp

```diff
@@ -45,9 +43,9 @@ Main::Main(MainOpt &opt)
   finfo = std::shared_ptr<ForwarderInfo>(new ForwarderInfo(this));
   finfo->teamid = main_opt.teamid;
 
-  for (int i1 = 0; i1 < opt.conversion_threads; ++i1) {
+  for (size_t i1 = 0; i1 < opt.ConversionThreads; ++i1) {
     conversion_workers.emplace_back(new ConversionWorker(
-        &conversion_scheduler, opt.conversion_worker_queue_size));
+        &conversion_scheduler, opt.ConversionWorkerQueueSize));
```

Comment:
`opt.ConversionWorkerQueueSize` is cast to `uint32_t` from `size_t`, use `static_cast` if intentional.